README.md

```diff
@@ -67,6 +67,11 @@ Update-Module
 
 This will automatically load the module after starting PowerShell 3.0.
 
+You can check the version of the installed PnP-PowerShell module with the following command (replace the ``v16`` with ``v15`` when working with the On-Premises version).
+
+```powershell
+Get-Module officedevpnp.powershell.v16.commands -ListAvailable | select Version
```

Comment:
Only small thing that I would change this to 
```powershell
Get-Module OfficeDevPnP.Powershell.* -ListAvailable | Select Version,Name
```
This is because that will pickup the v15 modules as well and people like me have both installed & others may have the v15 version not the v16 version